File: Services/nexus.api.members/Service/UserServices.vb

Imports nexus.web.auth.Data

Namespace nexus.web.auth
    Public Class UserServices
        Implements IUsers
        Private _dbcontext As DbContextData

        Public Sub New(dbContextData As DbContextData)
            _dbcontext = dbContextData
        End Sub
        Public Function GetAll() As IEnumerable(Of Users) Implements IUsers.GetAll
            Return _dbcontext.Users
        End Function

    End Class
End Namespace

File: Services/nexus.api.local/Service/UserServices.cs

using nexus.web.auth;
using nexus.web.auth.Data;

namespace nexus.web.auth
{
    public class UserServices : IUsers
    {
        private DbContextData _dbcontext;

        public UserServices(DbContextData dbContextData)
        {
            _dbcontext = dbContextData;
        }
        public IEnumerable<Users> GetAll()
        {
            return _dbcontext.Users;
        }

    }
}
